utilities/console.py

```
@@ -68,13 +73,14 @@ def __init__(
     @retry(wait_timeout=TIMEOUT_5MIN, sleep=TIMEOUT_10SEC)
     def connect(self):
         LOGGER.info(f"Connect to {self.vm.name} console")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)